versioned_docs/version-24.6.0/tutorials/load-launchpad-keystores.md

```diff
@@ -10,7 +10,7 @@ The Staking Launchpad tool is used to create validators that participate in the
 proof-of-stake network. The tool generates an encrypted keystore file containing the validator details.
 Load this keystore into Web3Signer to sign attestations and blocks with the validator details.
 
-This tutorial uses Teku and Web3Signer to run validators created on the `goerli` testnet.
+This tutorial uses Teku and Web3Signer to run validators created on the `holesky` testnet.
```

Comment:
Did they request the versioned docs be updated as well?

Reply:
will ask that on the issue: I assumed not as Besu updates were not made historically for similar global change